HISTORY.md

```diff
@@ -9,6 +9,7 @@
 
 ### Bug Fixes
 * fix deadlock with enable_pipelined_write=true and max_successive_merges > 0
+* Fix corruption in non-iterator reads when mmap is used for file reads
```

Comment:
BTW, I didn't mention PinnableSlice here because we use it internally in Get(), so people who don't use PinnableSlice directly could still be affected.